notification/messagestatus.go

```diff
@@ -42,9 +42,12 @@ func (stat *MessageStatus) wrap(ctx context.Context, n *namedSender) *MessageSta
 type MessageState int
 
 const (
-	// MessageStateActive should be specified when a message is still active.
-	// This includes things like remotely queued, ringing, or in-progress calls.
-	MessageStateActive MessageState = iota
+	// MessageStateSending should be specified when a message is sending but has not been sent.
+	// This includes things like ringing, or in-progress calls.
```

Comment:
As stated in another comment, this still includes remotely queued. Think of the remote queue as an implementation detail of the other service. From GoAlert's perspective, the message is on-its way (we've handed off control).
```suggestion
	// This includes things like remotely queued, ringing, or in-progress calls.
```